components/collapsible/collapsible.stories.js

```diff
@@ -1,9 +1,10 @@
+import { createTemplateFromVueFile } from '@/common/storybook_utils';
 import {
   DtCollapsible,
 } from './';
-import { createTemplateFromVueFile } from '@/common/storybook_utils';
 import CollapsibleMdx from './collapsible.mdx';
 import DtCollapsibleDefaultStory from './collapsible_default.story.vue';
+import DtCollapsibleVariantsStory from './collapsible_variants.story.vue';
 
 const argTypesData = {
```

Comment:
remove the old "content" slot from the argTypesData, it's making this weird uncategorized field show up in storybook

Reply:
Then add the new slots `contentOnCollapsed` `contentOnExpanded` and make their control 'text' and summary 'VNode' as the other slots below.

Reply:
Done